src/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+	<meta name="theme-color" content="#000000">
+	<!--
+      manifest.json provides metadata used when your web app is added to the
+      homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
+    -->
+	<link rel="manifest" href="%PUBLIC_URL%/manifest.json">
```

Comment:
where did this html template come from? it's referencing a manifest.json, which you don't have in your repo, so when I run the app locally I get this error "URIError: Failed to decode param '/%PUBLIC_URL%/manifest.json'".
it looks like the html from create-react-app. i'd change this (not sure if it's part of your template or a different npm setup package, but i'd just replace it with a standard html template with the root div in it for future projects.